server/helpers/Helper.js

```diff
@@ -4,12 +4,6 @@ class Helper {
    constructor(){
        dotenv.config();
        this.bcrpt = bcrpt;
-       this.isStrongPassword = password => {
```

Comment:
Why are you removing the strongPassword validation?

Reply:
because joe test it too, so it was somehow duplication